test/extensions/extensions_build_system.bzl

```diff
@@ -1,54 +1,55 @@
 load("//bazel:envoy_build_system.bzl", "envoy_benchmark_test", "envoy_cc_benchmark_binary", "envoy_cc_mock", "envoy_cc_test", "envoy_cc_test_binary", "envoy_cc_test_library", "envoy_py_test")
 load("@envoy_build_config//:extensions_build_config.bzl", "EXTENSIONS")
 
-def _apply_to_extension_allow_for_test(func, name, extension_name, **kwargs):
-    if not extension_name in EXTENSIONS:
-        return
+def _apply_to_extension_allow_for_test(func, name, extension_names, **kwargs):
+    for extension_name in extension_names:
+        if not extension_name in EXTENSIONS:
+            return
```

Comment:
iiuc if any of the extension names dont match then the test wont run

is there a situation where perhaps one of the names is listed in EXTENSIONS but another isnt, but we still want the test to run ?

should the logic not be more like

```
if extension_name in extension_names:
    func(name, **kwargs)
    # maybe return here if we dont want the test to run twice
```

Reply:
ok - ignore me - i think iuc now - the test should not run if any of the extensions are not listed